__test__/index.spec.mjs

```diff
@@ -0,0 +1,7 @@
+import test from 'ava'
+
+import { goodFences } from '../index.js'
+
+test('sum from native', (t) => {
+  goodFences(["packages", "shared"], "../tsconfig.paths.json", ".", "../client-web", undefined);
```

Comment:
Want to point this at the integration tests in this repo instead of client-web?

Reply:
also the api should return a result you can assert against:

result = goodFences(..)
expect(result).toMatchInlineSnapshot()

Reply:
I think you should be able to run this with an action that just runs:

```
yarn napi build
yarn jest
```

Reply:
Changed this to work with `tests/good-fences-integration` instead of client-web